### PR TITLE
security-package/issues/140: Link "Read instructions on how to generate signature" in Store-Configuration is incorrect.

### LINKED ISSUE #140: Link "Read instructions on how to generate signature" in Store-Configuration is incorect
### Preconditions (*)

1. Magento(`2.4-develop`) with security-package(`1.1-develop`) is installed

### Steps to reproduce (*)

1. Login as Admin
2. Go to _STORE > CONFIGURATION >SECURITY > Security.txt_
3. Click to open _Other Information_ section
4. Scroll down to the `Signature` and click the link "Read instructions on how to generate signature"

### Actual result (*)
- opens https://devdocs.magento.com/#/individual-contributors page, without any information or instruction about a signature

### Expected result (*)
- the expected result will be defined after or during https://github.com/magento/security-package/issues/138
- It could lead to the dev/merch docs
- it could lead to the [README.md](https://github.com/magento/security-package/blob/1.0-develop/Securitytxt/README.md)

### PR DESCRIPTION
**Pull request by engcom-Foxtrot**
…

### Fixed Issues (if relevant)

1. https://github.com/magento/security-package/issues/140: Link "Read instructions on how to generate signature" in Store-Configuration is incorrect.

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Author has signed the [Adobe CLA](https://opensource.adobe.com/cla.html)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by engcom-Delta** [comment]
:heavy_check_mark: QA passed

File: Securitytxt/Model/Config/Signature.php [hunk 1]
```diff
@@ -24,7 +24,7 @@ class Signature implements CommentInterface
      */
     public function getCommentText($elementValue): string
     {
-        return "<a href='https://devdocs.magento.com/' target='_blank'>
+        return "<a href='https://github.com/magento/security-package/blob/1.0-develop/Securitytxt/README.md' target='_blank'>
```
Comment (naydav) [inline Securitytxt/Model/Config/Signature.php]: Please, move the link to DI onfiguration

Comment (engcom-Foxtrot) [reply Securitytxt/Model/Config/Signature.php]: @naydav done.

**Comment by engcom-Delta** [comment]
:heavy_check_mark: QA passed
Result:
image

**Comment by lenaorobei** [comment]
@magento run all tests

File: Securitytxt/Model/Config/Signature.php [hunk 2]
```diff
@@ -24,7 +38,7 @@ class Signature implements CommentInterface
      */
     public function getCommentText($elementValue): string
     {
-        return "<a href='https://devdocs.magento.com/' target='_blank'>
+        return "<a href='{$this->instructionLink}' target='_blank'>
```
Comment (lenaorobei) [inline Securitytxt/Model/Config/Signature.php]: Please use `escapeUrl` and change to not to show anything if `instructionLink` is empty.
Make sure to use translation as well.

**Comment by lenaorobei** [comment]
@magento run all tests

**Comment by engcom-Delta** [comment]
:heavy_check_mark: QA passed
**Result:**
image

- Translated description for instruction link

image

**Comment by lenaorobei** [comment]
@magento run Functional Tests B2B

**Review by lenaorobei — APPROVED** [review]

**Comment by lenaorobei** [comment]
B2B test is random https://jira.corp.magento.com/browse/MC-34516

**Pull request merged**